js/map.js

```diff
@@ -13,19 +15,73 @@ window.map = (function () {
     x: 0,
     y: 0
   };
+  var isPageActive = false;
+  var data = [];
+  var errorTemplate = document.querySelector('#error').content.querySelector('.error');
+
+  function translateData(receivedData) {
+    receivedData[0].offer.title = 'Cozy nest for newlyweds';
+    receivedData[0].offer.description = 'Gorgeous townhouse in downtown Tokyo. Suitable for both tourists and businessmen. The house is fully equipped and has a fresh renovation.';
+    receivedData[1].offer.title = 'Small apartment near the park';
+    receivedData[1].offer.description = 'Small clean apartment on the edge of the park. Without the Internet and registration.';
+    receivedData[2].offer.title = 'A bench in the park';
+    receivedData[2].offer.description = 'Great bench in the middle of the park. Suitable for people who like to sleep in the fresh air.';
+    receivedData[3].offer.title = 'Imperial Palace in downtown Tokyo';
+    receivedData[3].offer.description = 'Wonderful palace in the old city center. Only for those who can afford the palace.';
+    receivedData[4].offer.title = 'Sweetest attic';
+    receivedData[4].offer.description = 'Small apartment in the attic. For the most demanding.';
+    receivedData[5].offer.title = 'Coffee lovers hangout';
+    receivedData[5].offer.description = 'We have all! Coffee, internet and more coffee';
+    receivedData[6].offer.title = 'Clear hut';
+    receivedData[6].offer.description = 'We have all nishtyak here. Stall around the corner. shops open for 24 hours. Come on! Internet connection not provided!';
+    receivedData[7].offer.title = 'Apartment in the center';
+    receivedData[7].offer.description = 'It is beautiful, light and cozy. It can accommodate 5 people. Coffee and cookies provided for free.';
+    receivedData[8].offer.title = 'Quiet apartment near the subway';
+    receivedData[8].offer.description = 'Apartment on the first floor. Neighbors are quiet. For anyone who can not stand the hustle and bustle.';
+    receivedData[9].offer.title = 'Cute nest for Anime fans';
+    receivedData[9].offer.description = 'Enjoy your time nad please do not disturb others.';
+    return receivedData;
+  }
+
+  function errorReceiveHandler(errorText) {
```

Comment:
Looks like you're using the same code as in form module. 
Let's create a separate module to handle messages and reuse it across.

Reply:
done